org.eclipse.jdt.core/search/org/eclipse/jdt/internal/core/search/indexing/SourceIndexerRequestor.java

```diff
@@ -525,7 +528,7 @@ public void popTypeName() {
 		try {
 			this.enclosingTypeNames[-1] = null;
 		} catch (ArrayIndexOutOfBoundsException e) {
-			e.printStackTrace();
+			Util.log(e);
```

Comment:
This code can't be normally executed, the code inside try will **always** throw an exception, the stack trace was only for debug purpose. So please simply revert this change or replace entire "try/catch" with `new Exception().printStackTrace()`.

Reply:
I took care of this one in https://github.com/eclipse-jdt/eclipse.jdt.core/pull/1374/files#diff-d9c0f9e135f1b04312aa1ed45b5af71e2f7d2822751572c81244b232216241c3